client/src/app/shared/progress-bar/progress-bar.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Subject, Observable } from 'rxjs';
+
+export enum ProgressBarEventType {
+  PROGRESS,
+  VISIBLE
```

Comment:
Little nit pick here: let's stick to the style of all other enums in the code base and make those PascalCase